coalib/output/ConsoleInteraction.py

```diff
@@ -529,7 +529,8 @@ def acquire_settings(log_printer, settings_names_dict, section):
     for setting_name, arr in sorted(settings_names_dict.items(),
                                     key=lambda x: (join_names(x[1][1:]), x[0])):
         value = require_setting(setting_name, arr, section)
-        result.update({setting_name: value} if value is not None else {})
+        result.update({setting_name: value} if value is not None else
+                      True if setting_name == 'use_spaces' else {})
```

Comment:
OK. I guess that not all the settings are true/false settings right ? If they are not what default setting should they be assigned. If they are true/false it makes everything even easier.